src/components/Layout/adminMobileNavBar.module.scss

```
@@ -0,0 +1,21 @@
+/*
+ * Fixing the bar to the viewport is the layout's job, not the component's —
+ * the same split the desktop sidebar uses (protectedLayout.less).
+ */
+.bar {
+    position: fixed;
+    // Above the page's own sticky header (z-index 900) — a paginator scrolling
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)